timeseries/src/autogluon/timeseries/predictor.py

```diff
@@ -512,28 +512,36 @@ def fit(
 
             In the above example, multiple versions of the DeepAR model with different values of the parameters
             "hidden_size" and "dropout_rate" will be trained.
-        hyperparameter_tune_kwargs : str or dict, optional
-            Hyperparameter tuning strategy and kwargs (for example, how many HPO trials to run). If ``None``, then
-            hyperparameter tuning will not be performed.
-
-            Currently, only HPO based on random search is supported for time series models.
-
-            Setting this parameter to string ``"random"`` performs 10 trials of random search per model.
-
-            We can change the number of random search trials per model by passing a dictionary as `hyperparameter_tune_kwargs`.
-            The dict must include the following keys
-
-            - ``"num_trials"``: int, number of configurations to train for each tuned model
-            - ``"searcher"``: currently, the only supported option is ``"random"`` (random search)
-            - ``"scheduler"``: currently, the only supported option is ``"local"`` (all models trained on the same machine)
+        hyperparameter_tune_kwargs : str or dict, default = None
```

Comment:
Nit: We usually use `str or dict, optional` to annotate the type `Optional[Union[str, dict]]`

Reply:
sure. but, how do we imply it's None by default?

Reply:
If the default value was not None, we would write 
```
hyperparameter_tune_kwargs : str or dict, default = "my_default_value"
```
while 
```
hyperparameter_tune_kwargs : str or dict, optional
```
implies that the default value is None. At least that's how it's used in the rest of TimeSeriesPredictor / TimeSeriesDataFrame documentation.

This is also a convention that Numpy uses - see, e.g., docs for `dtype` and `like` arguments [here](https://numpy.org/doc/stable/reference/generated/numpy.ones.html)